template.tpl

```diff
@@ -587,33 +615,53 @@ ___SANDBOXED_JS_FOR_SERVER___
 const createRegex = require('createRegex');
 const encodeUriComponent = require('encodeUriComponent');
 const eventData = require('getAllEventData')();
-const getTimestampMillis = require('getTimestampMillis');
 const getType = require('getType');
 const JSON = require('JSON');
 const logToConsole = require('logToConsole');
 const makeNumber = require('makeNumber');
 const makeString = require('makeString');
 const Object = require('Object');
 const sendHttpRequest = require('sendHttpRequest');
-const setResponseBody = require('setResponseBody');
 const setResponseHeader = require('setResponseHeader');
 const setResponseStatus = require('setResponseStatus');
 const sha256Sync = require('sha256Sync');
 const testRegex = require('testRegex');
+const getRequestHeader = require('getRequestHeader');
+const getContainerVersion = require('getContainerVersion');
```

Comment:
Move these two `const` declarations to correct positions in the API list (alphabetical order).